hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java

```diff
@@ -891,13 +896,25 @@ private String reconfDfsUsageParameters(String property, String newVal)
         for (FsVolumeImpl fsVolume : volumeList) {
           Map<String, BlockPoolSlice> blockPoolSlices = fsVolume.getBlockPoolSlices();
           for (BlockPoolSlice value : blockPoolSlices.values()) {
-            value.updateDfsUsageConfig(null, jitter);
+            value.updateDfsUsageConfig(null, jitter, null);
+          }
+        }
+      } else if (property.equals(FS_GETSPACEUSED_CLASSNAME)) {
+        Preconditions.checkNotNull(data, "FsDatasetSpi has not been initialized.");
+        Class<? extends GetSpaceUsed> klass = (newVal == null ? DU.class :
+                Class.forName(newVal).asSubclass(GetSpaceUsed.class));
```

Comment:
@singer-bin Thanks for updating the PR.
We may need to use `WindowsGetSpaceUsed.class` if it is windows os.
https://github.com/apache/hadoop/blob/4ef1d3eef9340b59f132421fba443dc456671ee4/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/GetSpaceUsed.java#L79-L83

Reply:
Thank you very much for your review, I will revise it right away. @tasanuma